dashboard/src/components/MainView/MainView.js

```diff
@@ -56,9 +57,9 @@ const MainView = () => {
               onPollutantSelect={id => {
                 const currPollutant = allPollutants.filter(pollutant => Number(pollutant.id) === Number(id));
                 setSelectedPollutant(currPollutant[0]);
+                setPollutantID(selectedPollutant ? selectedPollutant.id : null);
               }}
             />
-            {console.log(selectedPollutant.description)}
             <div className="pollutant-details-container">
               <PollutantDetails pollutant={selectedPollutant} />
```

Comment:
In the pollutant detail box, there is currently no useful data to show. Here's what we can do instead: you can search for each pollutant and include a brief description of them in the code. So, when a user clicks on a specific pollutant, they will see a short description of that pollutant. You can use Wikipedia as a great source for obtaining these descriptions. Additionally, we can add a "Learn more" link at the end of the pollutant detail box, which will direct users to the Wikipedia page of that particular pollutant.

Reply:
There are descriptions for a good amount of the pollutants, but some do not have any. If we want we can write the descriptions by hand though.

Reply:
Yes, we can do that. Feel free to let me know if you need help with that.

Reply:
If you could write up the descriptions, that would be a huge help. They are currently being pulled from this https://docs.openaq.org/reference/parameters_get_v2_parameters_get